src/System.Data.Common/tests/System/Data/ForeignKeyConstraintTest.cs

```diff
@@ -197,67 +197,37 @@ public void TestCtor5()
             // Create a ForeingKeyConstraint Object using the constructor
             // ForeignKeyConstraint (string, string, string[], string[], AcceptRejectRule, Rule, Rule);
             ForeignKeyConstraint fkc = new ForeignKeyConstraint("hello world", parentTableName, parentColumnNames, childColumnNames, AcceptRejectRule.Cascade, Rule.Cascade, Rule.Cascade);                                                                                                                            // Assert that the Constraint object does not belong to any table yet
-            try
-            {
-                DataTable tmp = fkc.Table;
-                Assert.False(true);
-            }
-            catch (NullReferenceException)
-            { // actually .NET throws this (bug)
-            }
-            catch (InvalidOperationException)
-            {
-            }
+
+            Exception ex = Assert.ThrowsAny<Exception>(() => fkc.Table);
+            Assert.True(ex is NullReferenceException || ex is InvalidOperationException);
+
 
             Constraint[] constraints = new Constraint[3];
             constraints[0] = new UniqueConstraint(column1);
             constraints[1] = new UniqueConstraint(column2);
             constraints[2] = fkc;
 
             // Try to add the constraint to ConstraintCollection of the DataTable through Add()
-            try
-            {
-                table2.Constraints.Add(fkc);
-                throw new ApplicationException("An Exception was expected");
-            }
+            //try
```

Comment:
Remove comment?

Reply:
Oh, whoops. I guess I could do that in a follow-up PR.

Reply:
I took care of it. Thanks.